src/java.base/share/classes/java/lang/invoke/X-VarHandleSegmentView.java.template

```diff
@@ -180,7 +179,7 @@ final class VarHandleSegmentAs$Type$s extends VarHandleSegmentViewBase {
         return convEndian(handle.be,
                           SCOPED_MEMORY_ACCESS.get$RawType$Volatile(bb.sessionImpl(),
                                   bb.unsafeGetBase(),
-                                  offset(bb, base, handle.alignmentMask)));
+                                   offsetNonPlain(bb, base, handle.alignmentMask)));
```

Comment:
The `offsetNonPlain` calls have an extra space:
```suggestion
                                  offsetNonPlain(bb, base, handle.alignmentMask)));
```

(this would’ve been avoidable if JDK used the more accessible tabs[^1] for indentation)

[^1]: https://github.com/prettier/prettier/issues/7475

Reply:
> The `offsetNonPlain` calls have an extra space:
> 
> (this would’ve been avoidable if JDK used the more accessible tabs[1](#user-content-fn-1-a99cebfdb5a6913c73e9a68c7c0e7c46) for indentation)
> ## Footnotes
> 
>     1. [Change `useTabs` to `true` by default prettier/prettier#7475](https://github.com/prettier/prettier/issues/7475) [↩](#user-content-fnref-1-a99cebfdb5a6913c73e9a68c7c0e7c46)

Part of the issue here is that this file is a template - so it's not a ".java" file, and as such, the IDE doesn't help with this stuff.

Reply:
Prettier is used on the CLI and in git hooks.